style.css

```diff
@@ -0,0 +1,264 @@
+  padding: 0.4rem 1.5rem;
+  color: #333;
+  text-transform: uppercase;
+  font-weight: bold;
+  background-color: #f1c40f;
+}
+
+.nav-link {
+  float: left;
+  display: block;
+  color: #000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.message-box {
+  margin: 1.5rem;
+  padding: 0.8rem;
+  text-align: center;
+}
+
+.message-box h1 {
+  font-size: 1.7rem;
+  margin-bottom: 1rem;
+}
+
+.message-box input {
+  display: block;
+  margin: auto;
+  width: 30%;
+  padding: 0.4rem;
+  margin-bottom: 1rem;
+}
+
+#features {
+  padding: 1rem;
+  margin-top: 3rem;
+  margin-bottom: 3rem;
+  display: flex;
+  flex-direction: column;
+}
+
+#features .feature {
+  display: flex;
+  justify-self: center;
+  align-items: center;
+}
+
+#features .feature i {
+  color: #ff8c00;
+}
+
+#features .feature div {
+  padding: 1rem;
+  margin-left: 3rem;
+}
+
+#features .feature div h2 {
+  font-size: 1.5rem;
+  margin-bottom: 0.5rem;
+}
+
+#how-it-work {
+  padding: 3rem;
+  margin: auto;
+  text-align: center;
+}
+
+#how-it-work #video {
+  width: 600px;
+  height: 340px;
+}
+
+#pricing {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 3rem 0.2rem;
+}
+
+.price-box {
+  border: solid #333 3px;
+  border-radius: 2px;
+  text-align: center;
+}
+
+.price-box:first-child {
+  margin-right: 1.2rem;
+}
+
+.price-box:last-child {
+  margin-left: 1.2rem;
+}
+
+.price-box h3 {
+  background-color: #ddd;
+  font-size: 1.05rem;
+  padding: 1rem;
+}
+
+.price-box h2 {
+  padding: 1rem;
+}
+
+.price-box p {
+  padding: 0 4rem 1rem;
+}
+
+.price-box .btn {
+  display: inline-block;
+  margin: 1rem;
+}
+
+/* FOOTER */
+#main-footer {
+  background-color: #ccc;
+  padding: 1rem 4rem;
+  margin-top: 2rem;
+  display: flex;
+  justify-content: space-between;
+  overflow: hidden;
+}
+
+#main-footer div a {
+  color: #333;
+  padding: 0.5rem 1rem;
+}
+
+#fcs {
+  text-decoration: underline;
+  color: #333;
+}
+
+/* TABLET AND SMARTPHONE */
+@media only screen and (max-width: 800px) {
+  #navbar {
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 1.5rem 0.3rem;
+  }
+
+  #navbar ul li {
+    padding: 1rem 1rem;
+  }
+
+  #features .feature {
+    flex-direction: column;
+  }
+
+  #features .feature div {
+    padding: 1rem;
+    margin: 0.5rem;
+    text-align: center;
+    margin-bottom: 2rem;
+  }
+
+  #how-it-work {
+    padding: 1rem;
+  }
+
+  #how-it-work #video {
+    width: 80%;
+    height: 30%;
+  }
+
+  #pricing {
+    flex-direction: column;
+  }
+
+  #pricing .price-box {
+    margin: 1rem;
+  }
+
+  .price-box:first-child {
+    margin-right: 0;
+  }
+
+  .price-box:last-child {
+    margin-left: 0;
+  }
+
+  #main-footer {
+    padding: 2.5rem 0.5rem;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  #main-footer .my-name {
+    margin: 1rem 0;
+  }
+}
```

Comment:
- [ ] Good job pointing out the removal of the excess brace 👍. However, it is good to include an explanation as to why a change is required with reasons why you think they should make the changes politely, as it will help the developer to see the importance of the change, and help them be a better developer. Please, let us edit this comment to be polite and to include why removing the extra brace is a good practice.

Reply:
Good job! Extra brace will likely give some errors during linters check. I think that removing the brace will give an error free program.  Suggestion: Curly braces - { and }  mark the beginning and end of the CSS code that specifies the style. All CSS rulesets must have opening and closing curly braces. The extra brace is going to stray.